controller/src/main/java/io/pravega/controller/store/stream/VersionedTransactionData.java

```diff
@@ -28,7 +28,8 @@
 @Data
 @AllArgsConstructor
 public class VersionedTransactionData {
-    public static final VersionedTransactionData EMPTY = new VersionedTransactionData(Integer.MIN_VALUE, new UUID(0, 0), null,
+    public static final VersionedTransactionData EMPTY = new VersionedTransactionData(Integer.MIN_VALUE,
```

Comment:
this seems like a formatting change...

Reply:
yes, i have made several formatting changes, particularly of the type where the line exceeded 120 chars.. it makes it very difficult to read the code so took care of that debt. have mentioned it in the description